aptos-move/move-examples/swap/sources/tests/liquidity_pool_tests.move

```diff
@@ -12,7 +12,7 @@ module swap::liquidity_pool_tests {
     fun test_e2e_volatile(lp_1: &signer, lp_2: &signer) {
         test_helpers::set_up(lp_1);
         let is_stable = false;
-        let (pool, tokens_2, tokens_1) = create_pool(lp_1, is_stable);
+        let (pool, tokens_1, tokens_2) = create_pool(lp_1, is_stable);
```

Comment:
Thank you! Can we also make the same adjustment on line 92?

Reply:
Why would the test have passed? The order of `tokens_1` and `tokens_2` does not matter here?